### DISCUSSION TITLE
Needs some time and a few pointers

### DISCUSSION THREAD

**Discussion by deererum** [community/community#57694] [Mobile]
Hello! Thanks for your interest in my clumsy doings here and thanks for asking feetback. I love new fantastic solutions in the network, but I know this from the interface side. I olso know Github for a couple of years, but I'm still learning this by testing with the code. It's not easy for a humanist;... Added to this a small language barrier. (I'm from Poland - right next to Ukraine).
I would like to familiarize myself with the environment of applications that enable the creation of small independent servers (e.g. on smartphones). Thanks to it, we can locate and exchange historical and sociological data as well as current issues related to human rights issues and efforts for peace in our region. There is also a strong need for a variety of corporate-independent messaging and thought-sharing platforms.

It would be nice if some of the experienced coders could give some pointers in this regard.
Have a nice day.
@deererum

**Comment by stevepopovich** [comment]
Hi @deererum ! I respect you reaching out, but I think this may not be quite the right forum. If you may, trying reddit.com, maybe stackoverflow.com or linkedin.com for places to connect with people and facilitate conversation that you are looking for.

This space is for bugs or feedback related to the native GitHub Mobile app for iOS or Android. Thanks again!
